pages/global-profile/views/index.jsx

```diff
@@ -0,0 +1,135 @@
+
+import Profile from '../../profile/read/views/profile';
+import Modules from '../../profile/read/views/modules';
+import RelatedTasks from '../../task/list/views/related-tasks';
+import AsruRoles from '../components/asru-roles';
+
+const formatDate = (date, format) => (date ? dateFormatter(date, format) : '-');
+
+class Index extends React.Component {
+
+  render () {
+
+    const model = this.props.model;
+    const isOwnProfile = this.props.isOwnProfile;
+    const hasEstablishments = !!(model.establishments || []).length;
+
+    const showEmail = this.props.asruUser || model.asruLicensing || this.props.isOwnProfile;
+    const showDob = (this.props.asruUser || this.props.isOwnProfile) && model.dob;
+
+    return <Fragment>
+      <Header title={`${model.firstName} ${model.lastName}`} />
+      { this.props.dedupe }
+      <dl>
+        {
+          showEmail && (
+            <Fragment>
+              <dt>Email:</dt>
+              <dd>
+                <a href={`mailto:${model.email}`}>{ model.email }</a>
+                {
+                  isOwnProfile && (
+                    <Fragment>
+                      {' | '}
+                      <Link page="account.updateEmail" label="Edit" />
+                    </Fragment>
+                  )
+                }
+              </dd>
+            </Fragment>
+          )
+        }
+        {
+          model.telephone && <Fragment>
+            <dt>Telephone:</dt>
+            <dd>
+              { model.telephone }
+              {
+                isOwnProfile && (
+                  <Fragment>
+                    {' | '}
+                    <Link page="account.update" label="Edit" />
+                  </Fragment>
+                )
+              }
+            </dd>
+          </Fragment>
+        }
+        {
+          showDob && (
+            <Fragment>
+              <dt>Date of birth:</dt>
+              <dd>
+                { formatDate(model.dob, dateFormat.medium) }
+                {
+                  isOwnProfile && (
+                    <Fragment>
+                      {' | '}
+                      <Link page="account.update" label="Edit" />
+                    </Fragment>
+                  )
+                }
+              </dd>
+            </Fragment>
+          )
+        }
+        {
+          this.props.asruUser && (
+            <Fragment>
+              <dt>Has login:</dt>
+              <dd>{ model.userId ? 'Yes' : 'No' }</dd>
+            </Fragment>
+          )
+        }
+      </dl>
+      {
+        !model.asruUser && (
+          <Fragment>
+            <h2><Snippet>pil.training.title</Snippet></h2>
+            {
+              model.certificates && model.certificates.length > 0
+                ? <Modules certificates={model.certificates} />
+                : <p><em><Snippet>pil.training.none</Snippet></em></p>
+            }
+          </Fragment>
+        )
+      }
+
+      {
+        model.asruUser && <AsruRoles />
```

Comment:
This feels like it's leaking way too much information to external users. Shouldn't this be only for ASRU users viewing the profile?

Reply:
not according to the ticket - this is just a list of asru roles, so if the profile you are looking at is an asru user it will say whether they are an inspector/LO

Reply:
I think it's fine to expose if they are an inspector or LO, but I'd be wary of exposing whether they have any other roles.

Can we limit the output to _only_ inspector/LO?

Reply:
ah, yeah good shout